tools/scripts/cn_metrics.py

```diff
@@ -43,60 +43,64 @@
 )
 PARSER.add_argument("kvs", help="kvs name")
 
-def full_tree(ybuf: Optional[Dict[str, Any]], opt):
+def full_tree(ybuf: Optional[OrderedDict[str, Any]], opt):
     if not ybuf or "info" not in ybuf:
         return
 
-    print("t ", end=""),
-    oids = ["oid1", "oid2"]
+    # Vector of initial column widths (i.e., compc, dgen, keys, ...)
+    #
```

Comment:
Drop extra comment line

Reply:
The general idea is to let the code breathe, bind the comment to the code following it, but the intervening blank line makes the code much easier to read versus when it's directly preceded by a comment.

Reply:
I don't see it the same way as you, but sounds good...